src/components/editor/TranscriptEditor.tsx

```
@@ -247,43 +249,118 @@ export function TranscriptEditor({
+                              word.deleted
+                                ? "text-muted-foreground/30 line-through scale-95"
+                                : "text-foreground hover:bg-muted/80",
+                              isActive &&
+                                "bg-primary text-primary-foreground shadow-md scale-110 z-10 font-medium",
+                            )}
+                            title={
+                              word.deleted
+                                ? "Right-click to restore"
+                                : "Click to seek, Right-click to delete"
+                            }
+                          >
```

Biome (2.4.15)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)